Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -544,6 +544,31 @@ private DeployResult GetResult(string id, string activeDeploymentId, bool isDepl
                 string repositoryRoot = _environment.RepositoryPath;
                 var perDeploymentSettings = DeploymentSettingsManager.BuildPerDeploymentSettingsManager(repositoryRoot, _settings);
 
+                string delayMaxInStr = perDeploymentSettings.GetValue(SettingsKeys.MaxRandomDelayInSec, false);
+                if (delayMaxInStr != null) {
+                    int maxDelay;
+                    if(!Int32.TryParse(delayMaxInStr, out maxDelay) || maxDelay < 0)
```

Comment:
I was thinking of using the default value for user input error: 
```
SCM_MAX_RANDOM_START_DELAY_IN_SECONDS = true
SCM_MAX_RANDOM_START_DELAY_IN_SECONDS = 10s
SCM_MAX_RANDOM_START_DELAY_IN_SECONDS = 123.45
...
```
in otherwords, user still wants the delay, but the format is off
if it is the convention to treat it as "null", I can surely change it back